README.md

```diff
@@ -47,7 +47,7 @@ inherit_gem:
 
 ## Testing
 
-```
+```bash
```

Comment:
Thanks for improving this project’s README!

One suggestion, since these commands might be run in other shells (`zsh`, `fish`, etc.):
```suggestion
```sh
```

Reply:
Dear @smockle ,

Thank you for your support. I updated. Please help me to review it.

Thanks,
Lam